eth/catalyst/api.go

```diff
@@ -93,6 +93,7 @@ var caps = []string{
 	"engine_exchangeTransitionConfigurationV1",
 	"engine_getPayloadV1",
 	"engine_getPayloadV2",
+	"taiko_engine_getPayloadV2", // CHANGE(taiko): added new method
```

Comment:
```suggestion
	"engine_getTaikoPayloadV2", // CHANGE(taiko): added new method
```
How about using the name `engine_getTaikoPayloadV2`?